code/game/machinery/computer/communications.dm

```diff
@@ -161,6 +161,12 @@
 			if (!message_index)
 				return
 			LAZYREMOVE(messages, LAZYACCESS(messages, message_index))
+		if ("emergency_meeting")
+			if(!(SSevents.holidays && SSevents.holidays[APRIL_FOOLS]))
+				return
+			if (!authenticated_as_silicon_or_captain(usr))
```

Comment:
silicons are gonna be able to call emergency meetings?

Reply:
yes direct blame to athath

Reply:
ai will save the round from the imposter